docs/docs/configuration.md

```diff
@@ -112,6 +112,10 @@ Defaults to `true`.
 Create the virtualenv inside the project's root directory.
 Defaults to `false`.
```

Comment:
Should have caught this earlier, we default to `unset` correct?

Reply:
No, the default is still `false`. (At least I haven't changed it). And in my opinion we shouldn't change this behavior.

Reply:
Well the issue is that with this change we break existing behaviour. Currently if default (`false`), and a `.venv` directory is found `poetry` uses it. This behaviour can be replicated by configuring default to to be `None`  with the new changes. And hence why I reckon we should update the documentation and default value to reflect this. Otherwise, we end up creating new venvs where this "misfeature" was relied upon.

Reply:
While it is correct that this PR changes the current behavior and it could be replicate by setting to `None`, I don't think we should change the default value. Before this PR the behavior was surprising to many people and now it behave like the most people would expect. So it's more like a fix and not like a new feature.

But it's definitely something that should be stated out clearly in the release notes.

Reply:
The problem with that is that the current users who a) have not explicitly ever set it to `true` or `false`, and b) relies on existing `.venv` directories in their projects will suddently end up creating new virtual environments after upgrade. I do not think this is acceptable. Changing the default however, ensures that this case is covered for. And there is no difference for the other scenarios.